ezomero/_posts.py

```diff
@@ -522,6 +531,135 @@ def post_roi(conn, image_id, shapes, name=None, description=None,
     return roi.getId().getValue()
 
 
+def post_table(conn, table, object_type, object_id, title="", headers=True):
+    """Create new table and link it to an OMERO object.
+
+    Parameters
+    ----------
+    conn : ``omero.gateway.BlitzGateway`` object
+        OMERO connection.
+    object_type : str
+       OMERO object type, passed to ``BlitzGateway.getObjects``
+    object_id : int
+        ID of object to which the new Table will be linked.
+    table : object
+        Object containing the actual table. It can be either a list of
+        row-lists or a pandas Dataframe in case the optional pandas dependency
+        was installed. Note that each column should be of a single type;
+        mixed-type columns will be ignored. Types supported: int, string,
+        float, boolean.
+    title : str, optional
+        Title for the table. If none is specified, a `Table:ID` name is picked,
+        with a random UUID. Note that table names need to be unique!
+    headers : bool, optional
+        Whether the first line of the `table` object should be interpreted
+        as column headers or not. Defaults to `True` and is ignored for pandas
+        Dataframes.
+
+
+    Returns
+    -------
+    TableFile_id : int
+        ID of newly created FileAnnotation containing the new Table.
+
+
+    Notes
+    -------
+    Currently not working with `across_groups` - the `OriginalFile` seems to
```

Comment:
Is this worth a question to the OME team before merging?

Reply:
I think it's worth pushing it as is and opening a new issue for that problem - we can reach out to them when working that issue

Reply:
(in fact I had already created issue #59 lol)